vaadin-checkbox-flow-parent/vaadin-checkbox-testbench/src/main/java/com/vaadin/flow/component/checkbox/testbench/CheckboxGroupElement.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.checkbox.testbench;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.vaadin.flow.component.checkbox.testbench.CheckboxElement;
+import com.vaadin.testbench.HasHelper;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+
+@Element("vaadin-checkbox-group")
+public class CheckboxGroupElement extends TestBenchElement
+        implements HasHelper {
+
+    /**
+     * Gets a list of all available options.
+     *
+     * @return a list of the options (visible text)
+     */
+    public List<String> getOptions() {
+        return getCheckboxes().stream().map(CheckboxElement::getLabel)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Gets all checkboxes which are part of this group.
+     *
+     * @return a list of all checkboxes in this group
+     */
+    public List<CheckboxElement> getCheckboxes() {
+        return $(CheckboxElement.class).all();
+    }
+
+    /**
+     * Selects the first checkbox matching the given text. If it is selected
+     * does nothing.
+     *
+     * @param text
+     *            the text of the option to select
+     */
+    public void selectByText(String text) {
+        Optional<CheckboxElement> checkbox = getCheckboxByText(text);
+        if (!checkbox.isPresent()) {
+            throw new NoSuchElementException(
+                    "No item with text '" + text + "' found");
+        }
+
+        checkbox.get().setChecked(true);
+    }
+
+    /**
+     * Unselects the first checkbox matching the given text. If it is unslected
+     * does nothing.
```

Comment:
```suggestion
     * Unselects the first checkbox matching the given text. Does nothing
     * if the item is not selected.
```

Applicable to other methods as well.